### DISCUSSION TITLE
iOS Companion app playback view

### DISCUSSION THREAD

**Discussion by Disincentivato** [blakeblackshear/frigate#14260] [General]
Hi,
in playback view the entire window often shifts up for about one centimeter and there is no way to get it back unless I rotate the phone horizontally and then vertically again.
Iphone 15 and HA Companion app. (all updated to the last version)

![IMG_7473](https://github.com/user-attachments/assets/4ac1e563-ffd0-4926-bbea-6e2915287cca)

**Comment by NickM-27** [comment]
This is a known issue with the companion app

**Reply by Disincentivato** [reply]
oh thank you for let me know.
